Learn_go/网络通信/learn03-template/server/main.go

```
@@ -0,0 +1,54 @@
+	}
+	s.ListenAndServe()
+}
+
+type handler1 struct{}
+
+func (h *handler1) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	t1, err := template.ParseFiles("F:/Go/Learn_go/网络通信/learn03-template/html/index.html")
+	if err != nil {
+		panic(err)
+	}
+	t1.Execute(w, "Hello MyHTML")
```

golangci-lint (1.64.8)

Error return value of `t1.Execute` is not checked

(errcheck)